src/glmfit.jl

```diff
@@ -579,6 +579,12 @@ function fit(::Type{M},
         throw(DimensionMismatch("number of rows in X and y must match"))
     end
 
+    wts isa FrequencyWeights ||
+        Base.depwarn("Passing weights as vector is deprecated in favor of explicitly using " *
+                    "`FrequencyWeights` from StatsBase. " *
+                    "Proceeding by coercing `wts` to `FrequencyWeights`.",
+                    :fit)
```

Comment:
Maybe enforce the deprecation warning?
```suggestion
                    :fit; force = true)
```

Reply:
That deprecation isn't particularly problematic (if you missed it, we will print an explicit error message for some time in 2.x I think), so I wouldn't force it. Users who want to keep using GLM 1.x without changing their code immediately should be able to do that IMO.